.github/workflows/main.yml

```diff
@@ -85,7 +98,7 @@ jobs:
     if: ${{ contains(fromJson(needs.decision.outputs.configuration).platforms, 'windows') }}
     uses: ./.github/workflows/windows.yml
     with:
-      profile: "release"
+      profile: ${{ fromJson(needs.decision.outputs.configuration).profile }}${{ fromJson(needs.decision.outputs.configuration).profile }}
```

Comment:
Looks like this `${{ fromJson(needs.decision.outputs.configuration).profile }}` was inadvertently doubled here?

Reply:
This is my touchpad, will fix it.